config/settings.yml.example

```diff
@@ -58,6 +58,15 @@
 # default values for https_port is 8443
 #:https_port: 8443
 
+# http server type configuration
+# A string that indicates the server type (possible values: 'webrick', 'puma').
+# Default value is 'webrick'
+#:http_server_type: 'puma'
+# https server type configuration
+# A string that indicates the server type (possible values: 'webrick', 'puma').
+# Default value is 'webrick'
+#:https_server_type: 'puma'
```

Comment:
I don't think we really need separate server_type setting for http and https and I think it's just misleading. One option should rule them all